packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -195,6 +195,7 @@ export default {
     },
 
     mounted () {
+        this.otp = ''; // Clear the field
```

Comment:
Should test be added to this? i.e on mounted it should not clear the otp out field.

Reply:
This field will be blank on first load due to the prop being new and empty, the reason for the clearing here is that when you rock back/fwd on the browser buttons we need to clear field ... and this cannot be done via a unit test afaik.